script/sync-crls

```diff
@@ -1,11 +1,14 @@
 #!/bin/bash
 
 # script/sync-crls: update the DOD CRLs and place them where authnid expects them
+set -e
+cd "$(dirname "$0")/.."
 
-rm -rf crl/ || true
+mkdir -p crl-tmp
+pipenv run python ./authnid/crl/util.py crl-tmp
 mkdir -p crl
-echo "Updating local CRLs..."
-pipenv run python ./authnid/crl/util.py crl
+rsync -rq crl-tmp/. crl/.
```

Comment:
I thought rsync was for copying files between two different servers -- what's the benefit of using it here?